[BUG] Wrong encoding detected for empty JSON response

**Describe the bug**

`requests 2.26.0` switched to using `charset_normalizer` by default under Python 3 (see: https://github.com/googleapis/python-cloud-core/issues/117).  After this change, a response constructed with an empty JSON body (`b"{}`) can no longer unmarshall to the empty dict in its `json` method.

**To Reproduce**

```python
>>> import charset_normalizer
>>> empty_json_response = b"{}"
>>> detected = charset_normalizer.detect(empty_json_response)
....nox/unit-3-6/lib/python3.6/site-packages/charset_normalizer/api.py:95: UserWarning: Trying to detect encoding from a tiny portion of (2) byte(s).
  warn('Trying to detect encoding from a tiny portion of ({}) byte(s).'.format(length))
>>> detected
{'encoding': 'utf_16_be', 'language': '', 'confidence': 1.0}
>>> decoded = empty_json_response.decode(detected["encoding"])
>>> decoded
'筽'
>>> import json
>>> json.loads(decoded)
/opt/Python-3.6.10/lib/python3.6/json/__init__.py:354: in loads
    return _default_decoder.decode(s)
/opt/Python-3.6.10/lib/python3.6/json/decoder.py:339: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <json.decoder.JSONDecoder object at 0x7fd0b5810860>, s = '筽', idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` beginning with
        a JSON document) and return a 2-tuple of the Python
        representation and the index in ``s`` where the document ended.
    
        This can be used to decode a JSON document from a string that may
        have extraneous data at the end.
    
        """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration as err:
>           raise JSONDecodeError("Expecting value", s, err.value) from None
E           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Expected behavior**
I expect to be able unmarshall `b"{}"` into an empty dict, `{}`.

**Desktop (please complete the following information):**
 - OS: Linux
 - Python version: 3.6, 3.7, 3.8, 3.9
 - Package version: 2.0.1

--- a/charset_normalizer/md.py
+++ b/charset_normalizer/md.py
@@ -56,7 +56,7 @@ class TooManySymbolOrPunctuationPlugin(MessDetectorPlugin):
     def feed(self, character: str) -> None:
         self._character_count += 1
 
-        if character != self._last_printable_char and character not in ["<", ">", "=", ":", "/", "&", ";"]:
+        if character != self._last_printable_char and character not in ["<", ">", "=", ":", "/", "&", ";", "{", "}", "[", "]"]:
             if is_punctuation(character):
                 self._punctuation_count += 1
             elif character.isdigit() is False and is_symbol(character):

--- a/tests/test_on_byte.py
+++ b/tests/test_on_byte.py
@@ -27,6 +27,18 @@ class TestBytes(unittest.TestCase):
             len(r.alphabets)
         )
 
+    def test_on_empty_json(self):
+
+        with self.subTest("Detecting empty JSON as ASCII"):
+            results = from_bytes(b"{}").best()
+            self.assertIsNotNone(
+                results.best()
+            )
+            self.assertEqual(
+                results.best().encoding,
+                "ascii"
+            )
+
     def test_bom_detection(self):
         with self.subTest('GB18030 UNAVAILABLE SIG'):
             self.assertFalse(

